index.js

```diff
@@ -0,0 +1,53 @@
+const booksArrays = [];
+let booksObject = {};
+let booksFromStorage = JSON.parse(localStorage.getItem('bookStore')) || [];
+
+//  function to delete book
+
+function deleteBook(button) {
+  const btnId = Number(button.getAttribute('data-id'));
+  const newBooks = booksFromStorage.filter((book) => book.id === btnId);
+  localStorage.setItem('bookStore', JSON.stringify(newBooks));
+  booksFromStorage = newBooks;
+}
```

Comment:
- [ ] Please refactor the localstorage configs in this delete method because, in the browser, it doesn't actually delete the book and display the others that remain from the first attempt. 🙏